format/FormatNumber.ts

```diff
@@ -188,7 +188,7 @@ export function fmtBillions(v: number, opts?: NumberFormatOptions): ReactNode  {
  * Render a quantity value, handling highly variable amounts by using units of millions (m) and
  * billions (b) as needed.'
  */
-export function fmtQuantity(v: number, opts?: NumberFormatOptions) {
```

Comment:
I think we should have an  explicit QuantityFormatOptions type that extends NumberFormatOptions and includes a `useMillions` and a `useBillions` flag.

Reply:
kk on it.